tools/lib/provision.py

```diff
@@ -175,6 +181,12 @@
         "postgresql-10-pgroonga",
         "postgresql-10-tsearch-extras",
     ],
+    "centos7": COMMON_YUM_DEPENDENCIES + [
+        "postgresql10-server",  # TODO add the source
+        "postgresql10",
+        "postgresql10-pgroonga",  # TODO add the source
+        # TODO tsearch-extras must be compiled from scratch
```

Comment:
Agreed, we should probably just take the block of code for cloning+building `tsearch_extras` that we have in our "advanced setup" docs, put it into a shell script, and call that as the way to support non-Debian platforms where we don't have a package.